### ISSUE TITLE
Question: Subscription / Disconnection

### ISSUE THREAD

**Issue by track02** [FreeOpcUa/python-opcua#713]
I'm currently running a FreeOpcUa server on a Raspberry Pi and connecting to it with a Qt Android Application (QtOpcUa + Open62541 Backend)

I seem to have some trouble monitoring nodes, the connection will just drop out after around 10 minutes of starting the monitor/subscription of a node.

I'm not claiming it's an issue with the server as other clients seem to be okay but I'm having a hard time debugging this and was just wondering if this is something you've came across before or would have any better ideas on how to diagnose.

I've enabled logging on my server and see an entry closing the connection to the client and cleaning up.

Any ideas would be greatly appreciated, I'll leave a similar message for the Open62541/QtOpcUa developers too.

Many thanks.

Edit: Talking with a QtOpcUa developer today, he thinks it may be something to do with renewing the secure channel, are there any settings / options for this in FreeOpcUa?

**Comment by oroulet** [comment]
Did you solve that? I have had clients connected for days, so I would be surprised if we had an issue there..

**Comment by track02** [comment]
No, wasn't able to solve this one, if I remember the connection to the client would drop after 10 minutes when you tried to interact with a connected server it wasn't just with subscriptions.
